### ISSUE TITLE
content.opf -  'Maude' should be 'Maud'

### ISSUE THREAD

**Issue by johanley** [standardebooks/l-m-montgomery_anne-of-green-gables#2]
The author's middle name is not spelled correctly in one spot. 'Maude' should be 'Maud'.

**Comment by acabal** [comment]
Sorry for the late reply, this should be fixed now!

**Comment by johanley** [comment]
Hi Alex,

Thanks for the correction.

I don't know if this interests you, but I have made a new transcription of
this novel, which has been accepted by gutenberg.org.

The reason why I did that is because of a rather large number of issues
with the old, original transcription at gutenberg.org (which was likely
done in the 1990s). I found about 1,000 differences between the
gutenberg.org text and the first edition of the novel. For more background,
here is the documentation.
<https://johanley.github.io/anne-of-green-gables/index.html>

On gutenberg.org the old transcription is here:
https://www.gutenberg.org/ebooks/45

And my newer transcription is here:
https://www.gutenberg.org/ebooks/64365

- John
